demo/ui/utils/config.py

```diff
@@ -0,0 +1,42 @@
+import tomllib
```

Comment:
I'll suggest writing UTs as well for these changes or you can take it in a different PR as well. Whatever suits you

Reply:
This feature is my own idea. Let’s first see if the community recognizes this concept😄